### PR TITLE
docs: MAINTAINERS file and LGTM configuration

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: .lgtm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvals = 1
+pattern = "(?i)LGTM"
+self_approval_off = true
```
Comment (jirikuncar) [inline .lgtm]: @drjova probably at the beginning it should be `false`.

Comment (drjova) [inline .lgtm]: changed to ```false```

**Comment by tiborsimko** [comment]
LGTM

**Comment by jirikuncar** [comment]
LGTM

**Pull request merged**